### PR TITLE
Bugfix/zcs 1098

### PR DESCRIPTION
**Pull request by shri314**
Please see - https://jira.corp.synacor.com/browse/ZCS-1098.

### CODE REVIEW

File: build.pl [hunk 1]
```diff
@@ -189,7 +189,7 @@ ()
       print "=========================================================================================================\n";
 
       Die( "Bad version '${GLOBAL_BUILD_RELEASE_NO}'", "$@" )
-         if ( ${GLOBAL_BUILD_RELEASE_NO} !~ m/^[0-9][0-9]*[.][0-9][0-9]*[.][0-9][0-9]*$/ );
+        if ( ${GLOBAL_BUILD_RELEASE_NO} !~ m/^[0-9][0-9]*[.][0-9][0-9]*[.][0-9][0-9]*$/ );
```
Comment (plobbes) [inline build.pl]: nit: change each ```[0-9][0-9]*``` to be ```[0-9]+``` as that would be equivalent and simpler, no?

File: instructions/bundling-scripts/zimbra-store.sh [hunk 2]
```diff
@@ -408,18 +391,40 @@ CreateDebianPackage()
...
+                         | sed -e 's/_[0-9].*//' \
+                         | grep zimbra-mbox- \
+                         | sed '1s/^/, /; :a; {N;s/\n/, /;ba}')";
+
+      cat ${repoDir}/zm-build/rpmconf/Spec/${currentScript}.deb \
+         | sed -e "s/@@VERSION@@/${releaseNo}.${releaseCandidate}.${buildNo}.${os/_/.}/" \
+               -e "s/@@branch@@/${buildTimeStamp}/" \
+               -e "s/@@ARCH@@/${arch}/" \
+               -e "s/@@MORE_DEPENDS@@/${MORE_DEPENDS}/" \
+               -e "s/^Copyright:/Copyright:/" \
```
Comment (plobbes) [inline instructions/bundling-scripts/zimbra-store.sh]: nit: this line seems unnecessary

File: instructions/bundling-scripts/zimbra-store.sh [hunk 3]
```diff
@@ -408,18 +391,40 @@ CreateDebianPackage()
...
+                       | xargs -n1 basename \
+                       | sed -e 's/-[0-9].*//' \
+                       | grep zimbra-mbox- \
+                       | sed '1s/^/, /; :a; {N;s/\n/, /;ba}')";
+
     cat ${repoDir}/zm-build/rpmconf/Spec/${currentScript}.spec | \
     	sed -e "s/@@VERSION@@/${releaseNo}_${releaseCandidate}_${buildNo}.${os}/" \
             	-e "s/@@RELEASE@@/${buildTimeStamp}/" \
+                -e "s/@@MORE_DEPENDS@@/${MORE_DEPENDS}/" \
             	-e "s/^Copyright:/Copyright:/" \
```
Comment (plobbes) [inline instructions/bundling-scripts/zimbra-store.sh]: nit: this line also seems unnecessary, but maybe I'm missing something

File: rpmconf/Install/Util/utilfunc.sh [hunk 4]
```diff
@@ -659,7 +659,10 @@ checkExistingInstall() {
     isInstalled $i
     if [ x"$PKGINSTALLED" != "x" ]; then
       echo "FOUND $PKGINSTALLED"
-      INSTALLED="yes"
+      if [ "$i" != "zimbra-memcached" ]
```
Comment (plobbes) [inline rpmconf/Install/Util/utilfunc.sh]: nit: please use ```; then``` on the same line as ```if```.

**Comment by shri314** [comment]
I have incorporated all review comments.

**Pull request merged**